hack/local-up-cluster.sh

```diff
@@ -202,11 +201,7 @@ function set_service_accounts {
 
 function start_apiserver {
     # Admission Controllers to invoke prior to persisting objects in cluster
-    if [[ -z "${ALLOW_SECURITY_CONTEXT}" ]]; then
-      ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,SecurityContextDeny,ServiceAccount,DenyEscalatingExec,ResourceQuota
-    else
-      ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,ServiceAccount,DenyEscalatingExec,ResourceQuota
-    fi
+    ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
i would also be okay with omitting it by default.  I'm writing E2Es around
it now and i would like to be able to them run in the google jenkins.

On Monday, October 5, 2015, Brian Grant <notifications@github.com> wrote:

> In hack/local-up-cluster.sh
> <https://github.com/kubernetes/kubernetes/pull/15035#discussion_r41220936>
> :
>
> > @@ -202,11 +201,7 @@ function set_service_accounts {
> >
> >  function start_apiserver {
> >      # Admission Controllers to invoke prior to persisting objects in cluster
> > -    if [[ -z "${ALLOW_SECURITY_CONTEXT}" ]]; then
> > -      ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,SecurityContextDeny,ServiceAccount,DenyEscalatingExec,ResourceQuota
> > -    else
> > -      ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,ServiceAccount,DenyEscalatingExec,ResourceQuota
> > -    fi
> > +    ADMISSION_CONTROL=NamespaceLifecycle,NamespaceAutoProvision,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
>
> We need a way to remove SecurityContextDeny. I'd be in favor of omitting
> it by default, in the same vein as enabling everything else by default in
> the absence of real, comprehensive security policy.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15035/files#r41220936>.
>